addon/mirage/factory.js

```diff
@@ -6,18 +24,11 @@ export default class Factory {
     Object.keys(properties).forEach(key => {
       const property = properties[key];
 
-      if (property.type === 'string') {
-        attrs[key] = (i) => `${key} #${i}`;
-      } else if (property.type === 'number') {
-        attrs[key] = 0;
-      } else if (property.type === 'date') {
-        attrs[key] = () => new Date();
-      } else if (property.type === 'boolean') {
-        attrs[key] = false;
-      } else if (property.type === 'object') {
-        attrs[key] = {};
-      } else if (property.type === 'array') {
-        attrs[key] = [];
+      if (Ember.isArray(property.type)) {
+        const [firstType] = property.type;
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'destructuring expression' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).